### PR TITLE
ports/stm32: Remove hard-coded MDMA channels.

### PR DESCRIPTION
**Pull request by iabdalkader**

### CODE REVIEW

File: src/omv/ports/stm32/sensor.c [hunk 1]
```diff
@@ -57,6 +57,17 @@ void ISC_SPI_IRQHandler(void) {
 }
 #endif // ISC_SPI
 
+#if defined(OMV_MDMA_CHANNEL_DCMI_0)
```
Comment (kwagyeman) [inline src/omv/ports/stm32/sensor.c:60]: The ISRs for this should never fire. So, this isn't needed. However, it should also be harmless to have.

File: src/omv/imlib/jpeg.c [hunk 2]
```diff
@@ -1012,8 +1013,12 @@ void JPEG_IRQHandler() {
 }
 
 void jpeg_mdma_irq_handler() {
-    HAL_MDMA_IRQHandler(&JPEG_MDMA_Handle_In);
-    HAL_MDMA_IRQHandler(&JPEG_MDMA_Handle_Out);
+    if (MDMA->GISR0 & (1 << OMV_MDMA_CHANNEL_JPEG_IN)) {
```
Comment (kwagyeman) [inline src/omv/imlib/jpeg.c:1016]: Doesn't the IRQ handler it's self check the stream ISR? I guess this saves the jump.

**Review by kwagyeman — COMMENTED** [review]
Don't see any issues.

File: src/omv/imlib/jpeg.c (hunk 2 above)
Comment (iabdalkader) [reply src/omv/imlib/jpeg.c:1016]: > I guess this saves the jump.

Exactly.

**Pull request merged**